program/app.py

```diff
@@ -15,9 +22,12 @@
 for prd in ['year','week']:
     image_files.append(topn_requests_donut(dframe, period=prd))
```

Comment:
this approach causes the logger to log twice as for the same function-- once for each chart.